docs/articles/vs-test-adapter/Adapter-Installation.md

```diff
@@ -4,43 +4,63 @@ uid: vstestadapterinstallation
 
-> [!WARNING]
-> The use of the VSIX extension manager is deprecated.  It still works, but at some time support for it will be removed from Visual Studio, and also newer versions of the adapter. We recommend you try to move to nuget based adapters.
+Assume you are at a solution root, and you either have a solution file, or have just been adding it with ```dotnet new sln```
 
-To install the NUnit Test Adapter using the Extension Manager, follow these steps:
+```cmd
+md Whatever.Test
+cd Whatever.Test
+dotnet new nunit
+cd ..
+dotnet sln add Whatever.Test\Whatever.Test.csproj
+```
 
-1. From within Visual Studio, select Tools | Extension Manager.
-1. In the left panel of the Extension Manager, select Online Extensions
-1. Locate (search for) the NUnit 3.0 Test Adapter in the center panel and highlight it.
-1. Click 'Download' and follow the instructions.
+And you're ready to go !
 
-Use the Extension Manager to ensure that the NUnit 3.0 Test Adapter is enabled.
+### Manually adding it as a package reference to your test projects
 
-## How to choose between Extension and NuGet package
+The recommended set of packages you should install is the framework, the adapter and the analyzer.  The adapter and the framework is required, the analyzer will help you, but is not required to make your project work.  The coverlet.collector is the recommended code coverage package you should go for.
 
-The use of the VSIX extension manager is deprecated.  It still works, but at some time support for it will be removed from Visual Studio, and also newer versions of the adapter.
+You should ensure you have the ```Microsoft.NET.Test.Sdk``` in there too, as it is also required.
 
-We recommend you try to move to nuget based adapters.
+1. Open your test project csproj file
+2. Add the necessary package references as shown in the snippet example below (the versions should be the latest ones, the ones below is what is current at the date of writing)
 
-The NuGet Package will apply to the solution, and will work for any other user too, as it follows the solution, but requires the user to have VS 2012 Update 1 or above. It will also work for any TFS 2012 Update 1 or above server build, including TF Service and requires no further installation.
+```xml
+  <ItemGroup>
+    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="16.11.0" />
+    <PackageReference Include="NUnit" Version="3.13.2" />
+    <PackageReference Include="NUnit3TestAdapter" version="4.1.0" />
+    <PackageReference Include="coverlet.collector" Version="3.1.0" />
+    <PackageReference Include="NUnit.Analyzers" Version="3.2.0" />
+  </ItemGroup>
+```
 
-If you are testing **.NET Core or .NET Standard** you must use the NuGet package and Visual Studio 2017 or newer. Visual Studio does not support running .NET Core tests using an extension. The extension will only work with the full .NET Framework.
+Note: *You don't need to add any nunit.console or any other runner package*
```

Comment:
We should probably change this to the note format seen elsewhere in the docs. Let me know if you need a reference to where we do it elsewhere 👍